public/server/db/af.json

```diff
@@ -0,0 +1,4 @@
+{
+  "definition": "As Fuck",
```

Comment:
This looks good, but I would remove the alternative meaning.

Reply:
sure